pkg/controller/registry/resolver/resolver.go

```diff
@@ -246,22 +261,26 @@ func (r *SatResolver) getSubscriptionInstallables(pkg string, current *Operator,
 		depIds = append(depIds, c.Identifier())
 	}
 
+	if len(depIds) == 0 {
+		return nil, fmt.Errorf("no operators found for %s", pkg)
```

Comment:
This bubbles up to an overall resolution failure, which breaks scenarios containing subscriptions that are already at latest.